src/http/HttpError.js

```diff
@@ -0,0 +1,24 @@
+import HttpStatus from "./HttpStatus";
```

Comment:
Should have used a full path, otherwise breaks on Node.js
Fix in 17ab90de0b507ff0c49c0a499523e5f95edb900a

Reply:
cc @thanhdanh Has this been addressed?

Reply:
It's fixed in the above commit and published